packages/circuits/package.json

```diff
@@ -4,7 +4,17 @@
   "license": "BSD-3-Clause-Clear",
   "version": "1.0.0",
   "dependencies": {
+    "circom-helper": "^0.3.5",
     "circomlib": "^2.0.5",
+    "circomlibjs": "^0.1.7",
+    "ffjavascript": "^0.2.57",
     "snarkjs": "^0.6.9"
+  },
+  "scripts": {
+    "circom-helper": "npx circom-helper -c ./circom.config.json -b ./build/test -p 9001",
+    "test": "jest"
```

Comment:
I try to consolidate all useful commands in Makefiles, but I'll take care of this!